homeassistant/components/climacell/__init__.py

```diff
@@ -229,15 +355,23 @@ def _get_cc_value(
 
         return items.get("value")
 
+    @property
+    def entity_registry_enabled_default(self) -> bool:
```

Comment:
Will some entities be disabled that used to be enabled for users now?

Reply:
No, this should use the same logic as before. Will review it later to make sure that statement is accurate

Reply:
My statement was accurate, nothing has changed here